apps/portal/src/content/docs/components/label.mdx

```diff
@@ -28,7 +28,7 @@ import { DocsPage } from "../../../components/docs-page";
 
     <div className="flex flex-col gap-4">
         <h5>With color</h5>
-        <Label htmlFor="password" color="disabled">Password</Label>
+        <Label htmlFor="password">Password</Label>
         <input id="password" type="password" />
     </div>
 
```

Comment:
Please add an example for `primary` variant of `Label`.

Reply:
Added